samples/IoTLabWeather/SQL/vw.ReducedObservations.sql

```diff
@@ -0,0 +1,55 @@
+USE [IoTLabWeather]
+GO
+
+SET ANSI_NULLS ON
+SET QUOTED_IDENTIFIER ON
+GO
+
+-- ==================================================================
+-- Author:		Jan Machat (Redcley LLC)
+-- Create date: 4 May 2016
+-- Description:	Reduced set of observations: mostly-null cols dropped.
+-- Copyright � 2016 by Microsoft Corporation. All rights reserved.
+-- ===================================================================
+ALTER VIEW [dbo].[ReducedObservations]
+AS
+	SELECT TOP 10000000
```

Comment:
Yes, I am sure that’s enough. After four weeks of data gathering we have 22k+ rows. As these observations are for NOAA-decreed times, we have 37 years to deal with that.  ;-)

Unless NOAA changes their mind, of course. :-P

j

From: Sean Kelly [mailto:notifications@github.com]
Sent: Tuesday, May 10, 2016 4:12 PM
To: Redcley/iot-insider-lab <iot-insider-lab@noreply.github.com>
Cc: Jan Machat <jan@redcley.com>; Author <author@noreply.github.com>
Subject: Re: [Redcley/iot-insider-lab] Added two sprocs and corrected a bug (#8)


In samples/IoTLabWeather/SQL/vw.ReducedObservations.sql<https://github.com/Redcley/iot-insider-lab/pull/8#discussion_r62769671>:

> +USE [IoTLabWeather]

> +GO

> +

> +SET ANSI_NULLS ON

> +SET QUOTED_IDENTIFIER ON

> +GO

> +

> +-- ==================================================================

> +-- Author:          Jan Machat (Redcley LLC)

> +-- Create date: 4 May 2016

> +-- Description:     Reduced set of observations: mostly-null cols dropped.

> +-- Copyright � 2016 by Microsoft Corporation. All rights reserved.

> +-- ===================================================================

> +ALTER VIEW [dbo].[ReducedObservations]

> +AS

> +     SELECT TOP 10000000

Only the top 10 million records? Are you sure that will be enough? 😃

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Redcley/iot-insider-lab/pull/8/files/269e35102ad49a5c50f68f000b7c51ea1e071d71#r62769671>